src/components/orator/js/Orator.js

```diff
@@ -909,7 +909,8 @@ var fluid_3_0_0 = fluid_3_0_0 || {};
             toggle: {
                 funcName: "fluid.orator.selectionReader.togglePlay",
                 args: ["{that}", "{arguments}.0"]
-            }
+            },
+            isWord: "fluid.textNodeParser.isWord"
```

Comment:
The regex `\S` in `isWord()` checks the existence of a character rather than a word. It's probably better to name it `containChars()`.